tests/Controller/ExportControllerTest.php

```diff
@@ -36,7 +38,7 @@ protected function setUp(): void
     public function testExportOfSingleProduct(): void
     {
         $product = $this->createVisibleTestProduct();
-        $this->enableFindologicPlugin($this->getContainer(), self::VALID_SHOPKEY);
+        $this->enableFindologicPlugin($this->getContainer(), self::VALID_SHOPKEY, $this->salesChannelContext);
```

Comment:
Dont add any sales channel context in these tests. The controller must figure out which sales channel to use, based on the given shopkey.

Reply:
@TheKeymaster  The sales channel was already a 3rd argument in the plugin helper trait before my changes. I am passing a sales channel because we need one to set the config as for testing it sets a given shopkey in the sales channel. Otherwise the trait has no way to check what sales channel it has.

Reply:
Please see [this comment](https://github.com/findologic/plugin-shopware-6/pull/131#discussion_r529667561).